common/views/components/TabNav/TabNav.tsx

```diff
@@ -49,38 +54,31 @@ const NavItemInner: ComponentType<SpaceComponentProps> = styled(Space).attrs(
       }
     }
   }
-
-  &.selected:after {
-    width: 100%;
-  }
 `;
 
-const NavItem = ({ link, text, selected, onClick }: SelectableTextLink) => (
-  <NextLink {...link} passHref>
-    <Space
-      v={{
-        size: 'm',
-        properties: ['padding-top', 'padding-bottom'],
-      }}
-      as="a"
-      className={classNames({
-        'plain-link': true,
-        block: true,
-      })}
-      onClick={onClick}
-    >
-      <NavItemInner
-        as="span"
-        h={{ size: 'm', properties: ['margin-right'] }}
-        selected={selected}
-      >
-        {text}
-      </NavItemInner>
-    </Space>
-  </NextLink>
+const NavItem = ({
+  id,
+  text,
+  selected,
+  color,
+  onClick,
+}: SelectableTextLink) => (
+  <NavItemInner
+    as="span"
+    h={{ size: 'm', properties: ['margin-right'] }}
+    v={{ size: 'm', properties: ['padding-top'] }}
+    selected={selected}
+    color={color}
+    onClick={() => onClick(id)}
+    role="tab"
+    tabIndex={selected ? 0 : -1}
+    aria-selected={selected}
```

Comment:
I feel like this has to be `ariaSelected` here in order to render as `aria-selected` in the DOM but I might have that wrong.

Reply:
I know what you mean I've seen that error in previous project, but it renders out good 👍
<img width="406" alt="Screenshot 2022-08-17 at 10 40 56" src="https://user-images.githubusercontent.com/110461050/185087381-2434c5f3-2d80-47f8-a32c-70142e7d49c4.png">